cli-ext/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "A service to generate Hasura action scaffolds",
   "main": "src/server.js",
   "scripts": {
-    "get-shared-modules": "rm -rf src/shared && cp ../console/src/shared ./src/shared -r",
```

Comment:
This was not working as the usage of the `cp` command was incorrect. 
I was running this on a macOS machine